src/components/common/InputDialog.tsx

```diff
@@ -14,7 +14,7 @@ function InputDialog(props: DialogProps) {
   const classes = useStyles();
 
   return (
-    <Dialog maxWidth="sm" {...props}>
+    <Dialog maxWidth={props.maxWidth || 'sm'} {...props}>
```

Comment:
I wonder if we have to have this change, because if value maxWidth is in props then it will be overridden as comes after the attribute. No?

But if we have to have it then I think it is better to write {props.maxWidth ?? 'sm'} (with nullish coalescing)